sbin/attest-enroll

```diff
@@ -453,6 +462,28 @@ cleanup() {
 tmp="$(mktemp -d)"
 trap cleanup EXIT
 
+# We might need a TPM for what should be software-only things...
+start_swtpm() {
+	local -i tries port
+	((port= 10240 + ( ($$ % (32768 - 10240) ) ) ))
+	mkdir "${tmp}/swtpm"
+
+	for ((tries=0; tries < 3; tries++, port+=2)); do
+		swtpm socket --tpm2							\
+			     --daemon							\
```

Comment:
`--daemon` might be dangerous here.

Reply:
Gaa, that requires possibly waiting a long time.  Ok, no `--daemon`.

Reply:
Ah, but it's just as complicated w/o `--daemon`.

Reply:
I went back to using lsof, but better.